### DISCUSSION TITLE
✅ "Around Offset" for Loop and nested carousel widgets.

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#22772] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team.

It would be really great if you add "Around Offset" feature to Loop and nested carousel widgets. 

### Describe the solution you'd like

This feature actually places the Offset equally on the left and right side of the carousel. Please see the image below.

![Around Offset](https://github.com/elementor/elementor/assets/88034902/bb4dd27a-2590-470b-a9d4-7055228a10cf)

Also the following link :

https://codepen.io/desandro/pen/emjrBm

**Comment by corgalore** [comment]
The loop carousel has left/right offset already. The catch is that you need to enable Infinite Scroll.

![image](https://github.com/elementor/elementor/assets/554935/64f8afe3-8f4c-4f4d-8151-6adcb33aef29)

**Comment by nicholaszein** [comment]
Hi @Michaelgimii!

@corgalore is correct. This option is already available as he described.

I'm closing this discussion as solved.

Kind regards
